obal/data/roles/build_package/tasks/copr_build.yml

```
@@ -56,9 +56,22 @@
   when:
     - build_package_wait|bool
     - copr_builds is changed
-  include_tasks: wait.yml
-  vars:
-    copr_build_ids: "{{ copr_builds['results'] | selectattr('builds', 'defined') | map(attribute='builds') | list | flatten }}"
+  block:
+    - name: Extract Copr builds
+      set_fact:
+        copr_build_ids: "{{ copr_builds['results'] | selectattr('builds', 'defined') | map(attribute='builds') | list | flatten }}"
+
+    - name: "Watch copr build(s)"
+      command:
+        "copr-cli {{ '--config ' + build_package_copr_config if build_package_copr_config is defined else '' }} watch-build {{ copr_build_ids | join(' ') }}"
+      ignore_errors: true
+      register: build_package_copr_status
+      changed_when: false
```

you can do:
```suggestion
      changed_when: false
      vars:
        copr_build_ids: "{{ copr_builds['results'] | selectattr('builds', 'defined') | map(attribute='builds') | list | flatten }}"
```

To save the `set_fact`

---

I fail, you taught me this already.